assets/js/blocks/cart-checkout/placeholder-shipping-methods.js

```diff
@@ -0,0 +1,22 @@
+export default [
+	{
+		label: 'Click & Collect',
+		value: 'collect',
+		schedule: 'Pickup between 12:00 - 16:00 (Mon-Fri)',
+		price: 'FREE',
+	},
+	{
+		label: 'Regular shipping',
+		value: 'usps-normal',
+		dispatcher: 'Dispatched via USPS',
+		price: '€10.00',
+		schedule: '5 business days',
+	},
+	{
+		label: 'Express shipping',
+		value: 'ups-express',
+		dispatcher: 'Dispatched via USPS',
+		price: '€50.00',
+		schedule: '2 business days',
+	},
+];
```

Comment:
Does this follow the same shape as the response from the shipping endpoint? If not, it'd be a good idea to use the same shape and types as the rest endpoint response so it's easier to swap in the actual endpoint responses later.

Reply:
It's not, but in #1468 I already replaced it so if it's ok I would leave it like it is in this pull and leave the update in #1468.

I will update the `totalItems` shape in this PR, though.

Reply:
Yup works for me!